packages/vsf-storyblok-module/components/global/RichTextResolver.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div v-html="html" />
+</template>
+
+<script>
+import { Blok } from 'src/modules/vsf-storyblok-module/components'
+import Vue from 'vue'
+
+export default {
+  name: 'RichTextResolver',
+  props: {
+    richText: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    html () {
+      return Vue.prototype.$storyblokClient.richTextResolver.render(this.richText)
```

Comment:
@benjick 
I don't really feel comfortable just assuming `Vue.prototype.$storyblokClient` will exist here. Is there a better way of obatining the `storyblokClient` object via an import instead?
I tried doing it like this:
```
import Storyblok from 'storyblok-js-client'
const Api = new Storyblok()
```
But it threw an error. I assume because the constructor needs some config/options.